test/e2e_node/container_restart_test.go

```diff
@@ -55,12 +55,11 @@ var _ = SIGDescribe("Container Restart", feature.CriProxy, framework.WithSerial(
 			}
 		})
 
-		ginkgo.AfterEach(func() {
-			err := resetCRIProxyInjector(e2eCriProxy)
-			framework.ExpectNoError(err)
-		})
-
 		ginkgo.It("Container restart backs off.", func(ctx context.Context) {
+			ginkgo.DeferCleanup(func() {
```

Comment:
Please clean up: either call `ginkgo.DeferCleanup` after the code which causes the need to clean up (= in the `BeforeEach` above) or move that code here (= remove the `BeforeEach`).

Reply:
Thanks, Acknowledged

Reply:
Thanks, Done ( Note , i am copying the code from https://github.com/kubernetes/kubernetes/pull/128889/files here to perform tests before InPlacePodVerticalScaling Beta revert, if and when #128889 is merged, those will be removed during rebase )